test/presubmit-tests-with-pipeline-deployment.sh

```diff
@@ -68,45 +68,25 @@ GCR_IMAGE_BASE_DIR=gcr.io/${PROJECT}/${PULL_PULL_SHA}
 TEST_RESULTS_GCS_DIR=gs://${TEST_RESULT_BUCKET}/${PULL_PULL_SHA}/${TEST_RESULT_FOLDER}
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null && pwd)"
 
+# Configure `time` command output format.
+TIMEFORMAT="[test-timing] It took %lR."
+
```

Comment:
maybe this comment should be left in a better place - 
We should set gcloud to point to $PROJECT. Otherwise it can point to a different project than the one you want to test running on. 

gcloud config set project my-project

Reply:
It's included in `test-prep.sh`, so it's fine.